soc/arm/nordic_nrf/nrf54h/soc.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file SoC configuration macros for the
+ *       Nordic Semiconductor nRF54H family processors.
+ */
```

Comment:
this can be dropped, this file is not taken by Doxygen

Reply:
Applied.